packages/next/build/webpack/loaders/next-middleware-ssr-loader/index.ts

```diff
@@ -29,13 +28,7 @@ export default async function middlewareRSCLoader(this: any) {
 
         import App from ${stringifiedAbsoluteAppPath}
         import Document from ${stringifiedAbsoluteDocumentPath}
-
-        let ErrorPage
-        try {
-          ErrorPage = require(${stringified500PagePath}).default
-        } catch (_) {
-          ErrorPage = require(${stringifiedAbsoluteErrorPath}).default
-        }
+        const errorMod = require(${stringifiedAbsoluteErrorPath})
```

Comment:
Can we use `import`?

Reply:
import will load the default which is the error page, i'm also overriding with the page config and other stuff